Taller152/src/paqueteuno/EscribirArchivo.java

```diff
@@ -5,12 +5,44 @@
  */
 package paqueteuno;
 
+import java.util.Formatter;
 import paquetetres.*;
 
 /**
  *
  * @author reroes
  */
 public class EscribirArchivo {
+    public static void ejercicioo() {
+        Formatter archivo = null;
+        try {
+            String ruta = "/data/archivo1.txt";
+            String[] paises = {"Ecuador", "Colombia", "Perú", "Venezuela", 
+                "Chile", "Bolivia"};
+            String[] capitales = {"Quito", "Bogotá", "Lima", "Caracas", 
+                "Santiago", "La Paz"};
+            int[] poblacion = {17643054, 51152371, 33050325, 28435940, 19116201,
+                11673021};
+            String cadena = "";
+            archivo = new Formatter("data/archivo1.txt");
+            for (int x = 0; x < paises.length; x++){
+                archivo.format("%s %s, con capital %s tiene una poblacion"
+                        + " de %d\n", cadena, paises[x], 
+                        capitales[x], poblacion[x]);
+            }
+        }catch (Exception e) {
+            System.err.printf("Error, revise: %s\n", e.toString());
+
+        }finally{
+            archivo.close();
+        }
+       
+    }
+
+    static class escribirArchivo {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Class names should start with an upper case letter (NM_CLASS_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-class-names-should-start-with-an-upper-case-letter-nm-class-naming-convention).
Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML).